Continue company signup

alanhoskins opened the pull request:
[TICKET](https://artistguide.atlassian.net/browse/TICKET)

**Description:**

Continue the company signup. Currently there is a lot of commented out validation/error handling that I am working on in another PR. 


**PR Checklist:**

- [ X] If you haven't made changes to `package.json`, `package-lock.json` is not present in this PR
- [X ] You have tested these changes locally by running the app and letting it build successfully
- [ X] Eslint ran successfully upon commit and `--force` was not used to force this PR

_Make sure you **squash and merge** rather than simply merging all commits at once_.

At least **1** PR approval is required by a Tech Lead to be merged. **2** approvals total are required.

bestfoodalex commented on src/components/SignUp/Company/Details.tsx:
```diff
@@ -0,0 +1,167 @@
+
+export default CompanyDetails;
+
+const neighborhoods = [
```
We might want to move this to a utils file of some kind - might be useful elsewhere in the app?

alanhoskins replied:
Can do. I was debating a utils file or a db call just in case neighborhoods were added/removed in the future.

bestfoodalex commented on src/components/SignUp/Company/Photo.tsx:
```diff
@@ -0,0 +1,99 @@
+  font-style: italic;
+  font-weight: 500;
+  line-height: 14px;
+  letter-spacing: 0.4000000059604645px;
```
I'd truncate this greatly lol - not sure if we even need to specify letter spacing on this?

alanhoskins replied:
Haha. I didn't even notice. I yanked this from the design in Figma.

bestfoodalex commented on src/components/SignUp/SignUpFooter.tsx:
```diff
@@ -56,7 +56,8 @@ const SignUpFooter: React.FC<{
 
   const nextButtonAction = async (navNext: boolean, currStep: string) => {
-    if (currStep === 'privacy') {
+    console.log('currentStep', currentStep);
```
You'll probably want your own footer component for company signup to avoid complexity

alanhoskins replied:
I was thinking this as well. Will move forward with that.

bestfoodalex commented on src/genericComponents/Button.tsx:
```diff
@@ -7,27 +7,35 @@ const Button = (props: any) => {
-  line-spacing: 15;
-  padding: 16px 24px;
   text-transform: uppercase;
+  box-shadow: 0px 0px 8px 4px #0000000d;
```
Note the box shadow property already on this component

alanhoskins replied:
Missed that.

bestfoodalex commented on src/genericComponents/Button.tsx:
```diff
@@ -7,27 +7,35 @@ const Button = (props: any) => {
   text-transform: uppercase;
+  box-shadow: 0px 0px 8px 4px #0000000d;
+  height: 40px;
+  left: 754px;
```
Remove the absolute positioning properties

bestfoodalex commented on src/genericComponents/Button.tsx:
```diff
@@ -7,27 +7,35 @@ const Button = (props: any) => {
+  font-size: 14px;
+  font-weight: 700;
+  line-height: 16px;
+  letter-spacing: 0.1em;
```
And this seems to be a bit small

alanhoskins replied:
Again just from the figma design. Can adjust or put back.

bestfoodalex commented on src/genericComponents/Dropdown.tsx:
```diff
@@ -1,31 +1,51 @@
         onChange={onChange}
+        placeholder="Choose one..."
+        style={{
+          border: '1px solid #ced4da',
```
We'll want to grab colors from our color variables even in components like this

bestfoodalex commented on src/genericComponents/Input.tsx:
```diff
@@ -132,40 +158,22 @@ const InputField = (props: any) => {
-        <></>
-      )}
+      </CAGFormGroup>
+      {hasError && <CAGError>{errorMessage || ErrorMessage.Default}</CAGError>}
```
You'll want to run through the individual sign up to make sure this doesn't have any weird side effects there, but looks like a fine update

alanhoskins commented:
I think I addressed the issues. I upgrade react-bootstrap too as I was hitting an issue. I went through the site and individual signup and everything looked fine. Let me know if you want me to revert back.
